app/routes/invitations.rb

```diff
@@ -3,11 +3,11 @@ module Routes
     class Invitations < Core
       post '/teams/:slug/invitations' do |slug|
         please_login
+        valid_invitee = params[:usernames].delete(current_user.username)
         only_for_team_managers(slug, "You are not allowed to add team members.") do |team|
-          team.invite_with_usernames(params[:usernames], current_user)
+          team.invite_with_usernames(valid_invitee, current_user)
```

Comment:
I believe that the change in teams.rb from this commit actually does just that. Now when the team object is created initially by the user submitting the form on exercism.io/teams/new, then new code removes current _user from the array that is taken as a parameter during the creation of the team object. 

Once the creator of the team (current_user) is no longer in the Team object, the invitations can be sent to the whole team but the creator will not receive one.

I'm not sure if down the line there will be other uses it this particular team object, and whether the fact that the creator is not in there would present a problem. But I didn't see any other uses of this object besides it creating the team in the database and sending out the invitations.